bedrock/firefox/templates/firefox/enterprise/index.html

```diff
@@ -23,75 +25,64 @@
 
 {% block content %}
 <main role="main">
-  <section class="intro content-section">
-    <div class="content">
-      <div class="copy">
-        {{ high_res_img('logos/firefox/logo-quantum.png', {'alt': 'Firefox', 'width': '100', 'height': '100', 'class': 'logo'}) }}
-          <h1>{{ _('Firefox Quantum for Enterprise') }}</h1>
-          <p>{{ _('Speed up your business by giving employees a managed version of the super-fast new Firefox. <em>Now with Windows Group Policy support.</em>') }}</p>
-          <div class="copy-cta">
-            <a href="{{ url('firefox.enterprise.signup') }}" class="button button-green qa-sign-up">{{ _('Find Out More') }}</a>
-          </div>
-      </div>
-    </div>
-  </section>
+  {% call hero(
+    title=_('Firefox Quantum for Enterprise'),
+    desc=_('Speed up your business by giving employees a managed version of the super-fast new Firefox. <em>Now with Windows Group Policy support.</em>'),
+    class='mzp-has-image mzp-t-dark mzp-t-product-firefox t-enterprise',
+    include_cta=True,
+    image_url='firefox/enterprise/fx-browser-img.svg',
+    heading_level=1
+  ) %}
+  <a href="{{ url('firefox.enterprise.signup') }}" class="mzp-c-button mzp-t-product">{{ _('Find Out More') }}</a>
```

Comment:
This link has a functional test associated with it that relies upon the CSS class `qa-sign-up`. We should retain that class here in order for the test run to pass.